index.html

```diff
@@ -86,5 +86,197 @@ <h1>I'm Stephen Glad to see you</h1>
         </div>
       </section>
     </main>
+    <section class="project-section">
```

Comment:
- [ ] Good job on using Flexbox to style the project cards 👍 As stated in the requirements kindly use `grid` for the big container of the works section to demonstrate the ability to work with both 💯